packages/pds/tests/admin/get-repo.test.ts

```diff
@@ -91,6 +91,40 @@ describe('pds admin get repo view', () => {
     expect(triage).toEqual({ ...admin, email: undefined })
   })
 
+  it('includes emailConfirmedAt timestamp', async () => {
+    const { data: beforeEmailVerification } =
+      await agent.api.com.atproto.admin.getRepo(
+        { did: sc.dids.bob },
+        { headers: network.pds.adminAuthHeaders() },
+      )
+
+    expect(beforeEmailVerification.emailConfirmedAt).toBeUndefined()
+    const timestampBeforeVerification = Date.now()
+    const bobsAccount = sc.accounts[sc.dids.bob]
+    const verificationToken = await network.pds.ctx.services
+      .account(network.pds.ctx.db)
+      .createEmailToken(sc.dids.bob, 'confirm_email')
+    await agent.api.com.atproto.server.confirmEmail(
+      { email: bobsAccount.email, token: verificationToken },
+      {
+        encoding: 'application/json',
+        headers: {
+          authorization: `Bearer ${bobsAccount.accessJwt}`,
+        },
```

Comment:
I think you can use `sc.getHeaders(sc.dids.bob)` here rather than manually grabbing Bob's token and building the header.